api/app/Http/Middleware/AuditQueryMiddleware.php

```diff
@@ -28,11 +28,11 @@ public function __construct(LoggerInterface $logger)
     public function handle(Request $request, Closure $next)
     {
         $user = $request->user();
+        $referer = request()->headers->get('referer');
         if (! is_null($user) && $user->hasRole('platform_admin')) {
-            $message = 'GraphQL request from platform admin user ['.$user['email'].']';
+            $message = 'Request from platform admin, '.$user['email'].', referer: '.$referer.',';
             $this->logger->info(
-                $message,
-                $request->json()->all()
+                $message.' '.json_encode($request->json()->all())
```

Comment:
I think the `json_encode` is unnecessary.  The array from the `all` function is just going to be cast back to a string again.
```suggestion
                $message.' '.($request->json()->all())
```

Reply:
Looks like json encode is necessary to typecast the array

The above suggestion throws below error 
```
message: "Array to string conversion", exception: "ErrorException",…}
exception
: 
"ErrorException"
file
: 
"/home/site/wwwroot/api/app/Http/Middleware/AuditQueryMiddleware.php"

```

Reply:
Oh, you're right!  I'd still like to avoid parsing the JSON to a PHP array then encoding it back to a string again.  How about this?
```php
$message.' '.$request->getContent()
```
It looks like that's what the `json` function calls before decoding it.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/40115954-101c-4357-8653-01df771f271b)

Reply:
getContent() works! 🙌